test/integration/guest_returns_to_tribe_after_required_login_test.rb

```diff
@@ -0,0 +1,51 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.